code/modules/mob/living/basic/vermin/mouse.dm

```diff
@@ -116,7 +116,7 @@
 	. = ..(TRUE)
 	// Now if we were't ACTUALLY gibbed, spawn the dead mouse
 	if(!gibbed)
-		var/obj/item/food/deadmouse/mouse = new(loc, src)
```

Comment:
This just paves over the bug, it's supposed to be passing src to new, but the `/new` / `/initialize` overrides are wrong

![image](https://github.com/tgstation/tgstation/assets/51863163/a8120b97-aa46-4910-977d-57b08eed431e)

Reply:
This is fine but we really shouldn't have a `/New` override on an atom anyways, lol.